src/editor/render.js

```diff
@@ -33,13 +33,14 @@ function insertAfter(node, nodeToInsert) {
     }
 }
 
-export const ZERO_WIDTH_SPACE = "\u200b";
+// this is a BOM marker actually
```

Comment:
It might be good to expand on why we're using BOM instead.

Reply:
I was worried copying text out of the editor would contain invisible characters. On a first test, they *seem* to be filtered out, but this could be heavily platform dependent. As a precautionary measure, I chose the character that slate also uses. I'll update the comment accordingly.